pkg/machines/libvirt.es6

```diff
@@ -812,6 +850,7 @@ function startEventMonitor(dispatch, connectionName) {
         .fail(ex => {
             // this usually happens if libvirtd gets stopped or isn't running; retry connecting every 10s
             console.log("virsh event failed:", ex);
```

Comment:
virsh process can fail or be killed here, but is this console.log still needed?

Reply:
I'm fine with dropping it; I wanted to keep it in after landing the `virsh event` stuff, but I figure it's stable enough now.

Reply:
ok, I will remove it then